### ISSUE TITLE
Configuring TFLite(Android) to use all CPU's?

### ISSUE THREAD

**Issue by zhonghuabao1** [tensorflow/tensorflow#18770]
### System information
- **Have I written custom code (yes)**:
- **OS Platform and Distribution (Android Things)**:
- **TensorFlow installed from (source)**:
- **TensorFlow version (1.8.0-rc0)**:
- **Python version**:  2.7
- **Bazel version (0.12.0)**:
- **GCC/Compiler version (5.4.0)**:
- **CUDA/cuDNN version**: 9.1/7.1
- **GPU model and memory**: GTX1080 8G
- **Exact command to reproduce**:

### Describe the problem
I use Raspberry Pi3 + Android Things + TFLite + MobileNet to make a image classifier sample, base this [Android Things Image Classifier](https://codelabs.developers.google.com/codelabs/androidthings-classifier/#4) tutorial.
- I monitor the cpu by `adb shell top` and find it only use 1 CPU with quant model when inference, and 2 CPU with unquant model.
  So how to configuring TFLite(Android) to use all CPUs?

- Another question, maybe impact the question above, on the[ introduction of TFLite](https://www.tensorflow.org/mobile/tflite/), it say:
  > On select Android devices, the Interpreter will use the Android Neural Networks API for hardware acceleration, or default to CPU execution if none are available.

  You know, Raspberry has a GPU(Broadcom VideoCore IV), so how to use it? has it been used?

### Source code / logs
[Android Things Image Classifier](https://codelabs.developers.google.com/codelabs/androidthings-classifier/#4) tutorial.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @shivaniag: It has been 16 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @shivaniag: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @shivaniag: It has been 64 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @shivaniag: It has been 79 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by shashishekhar** [comment]
@zhonghuabao1 : You can try using Interpreter#SetNumThreads: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/lite/java/src/main/java/org/tensorflow/lite/Interpreter.java#L231 to change the number of threads.

Do let me know if you still need help.

**Comment by zhonghuabao1** [comment]
thanks for you ans, I have tried SetNumThreads a month ago yet. I fond it work with quantis models but not used with nonquantis models, such as MobileNet. 
I have not time to test it more recently, do you meet the problem? is it a bug?

**Comment by shashishekhar** [comment]
@zhonghuabao1 : you are correct, this was due to a bug, which we have recently fixed: see https://github.com/tensorflow/tensorflow/commit/2bfc7957dada57c1eb8491e04dac70d16b4369ac for the change, now it should work for non-quantized as well.
